Feature/data infra deployment doc update (#375)
Ensono/amido.github.io: docs/workloads/azure/data/requirements_data_azure.md

denoting variables required at the very outset of the project, and 'After Core Infrastructure
Deployment', referring to variables required after the fundamental infrastructure has been deployed.

> Please note that the variable group named `amido-stacks-euw-de-env-network` is only required if you want to provision infrastructure within a private network.

Reviewer: ```suggestion
ℹ️ The variables under `amido-stacks-euw-de-env-network` are only required if you want to provision the infrastructure within a private network.
```
